src/client/backup/reset/reset.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styles from './reset.scss'
+import {PrimaryButton} from '../../common/forms/Button'
+import {iBin} from '../../common/fonts/Icons'
+import {Messages, MessagesType} from '../../common/Messages'
+export function Reset() {
+    const messageType = MessagesType.ERROR
+    const messages = [ 'Reset your Nevergreen configuration back to defaults. Please note, resetting your configuration can not be undone!' ]
+    const resetConfiguration = () => {
+        window.localStorage.clear()
+        window.location.reload()
+    }
+    return(
+        <div>
```

Comment:
This should be wrapped in an actual [`Container`](https://github.com/build-canaries/nevergreen/blob/master/src/client/common/Container.tsx)